src/client/src/orangehrmAdminPlugin/pages/localization/LocalizationConfiguration.vue

```diff
@@ -115,10 +115,14 @@ export default {
   methods: {
     onSave() {
       this.isLoading = true;
-      this.http.http
-        .put('api/v2/admin/localization', {
-          language: this.configuration.language?.id,
-          dateFormat: this.configuration.dateFormat?.id,
+      this.http
```

Comment:
any special reason for this?

Reply:
refactored this code to use `request` method instance of accessing internal axios from api service instance